projects/igniteui-angular/src/lib/grids/common/crud.service.ts

```diff
@@ -91,16 +91,39 @@ export interface IgxAddRowParent {
 export class IgxCell {
     public primaryKey: any;
     public state: any;
+    public pendingValue: any;
 
     constructor(
         public id,
         public rowIndex: number,
         public column,
         public value: any,
-        public editValue: any,
+        public _editValue: any,
         public rowData: any,
         public grid: GridType) {
         this.grid.validation.create(id.rowID, rowData);
```

Comment:
@MayaKirova There's one odd case where the grid public `updateCell` API creates a new `IgxCell` with a different edit value and the form control will be out of sync here. That's actually not new it seems, however, with these changes the so will be the new value used by the API call (it just uses the args) to perform the update.
That's fortunately for blur only and could be logged separately, if we consider it still a minor use, though it's API that should work regardless.
Ideally, we'd initialize the form control with the edit value regardless instead of setting it additionally.